src/main.js

```diff
@@ -5,6 +5,7 @@ import './components/header/header.css';
 import './components/footer/footer.css';
 import Header from './components/header/header';
 import Footer from './components/footer/footer';
+import MovieComponent from './components/movie-component/movie-component';
```

Comment:
czemu dodajesz ten komponent do maina? Używamy go tylko w zakładce: `/pages/films/`

Reply:
On juz byl dodany do main.js, ja tylko zmienilam nazwe pliku bo sie npm build nie robil przez to